docs/input/docs/getting-started/principles.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
I feel like a lot of the content here should go into `Examples`. In technical core principles I would rather like to see principles, for example:

* A `Maybe` variable can never be `null`
* Else-Case et al

Reply:
I changed the title to core principles and moved implementation details to this section. I still believe that a short introduction should be given here.